aea/cli/init.py

```diff
@@ -93,8 +94,8 @@ def do_init(
     :param default_remote_registry: Default remote registry
     :param ipfs_node: URL to ipfs node
     """
-    config = get_or_create_cli_config()
-    if reset or config.get(AUTHOR_KEY, None) is None:
```

Comment:
This won't work since we have introduced the schema validation for cli config and value of `author` cannot be `None`. If we still want to use this logic we will have to update the validation schema to allow author to be `None` which is basically back to the square one.